pki/src/main/java/org/hyperledger/besu/pki/cms/CmsValidator.java

```diff
@@ -178,4 +178,18 @@ private boolean isCertificateTrusted(
       throw new RuntimeException("Error validating certificate chain", e);
     }
   }
+
+  private Optional<CertStore> loadCRLs(final KeyStoreWrapper truststore) {
```

Comment:
Noticed there is CRLUtil that does something similar to this function, should that be used instead or this replaces that one?

Reply:
CRLUtil is used when loading the CRL pem file. Not for the truststore. They are similar but not the same.